### ISSUE TITLE
error undefined is not an object (evaluating '_reactNative.Keyboard.addListener') 

### ISSUE THREAD

**Issue by danielCrespo01** [APSL/react-native-keyboard-aware-scroll-view#65]
I am trying to implement the library in tcombform this is my code:

`<KeyboardAwareScrollView>
<Form
ref = "LoginForm"
type = {} loginForm.type
options = {} loginForm.options
/>
</ KeyboardAwareScrollView>`

I generated the error:
undefined is not an object (Evaluating '_reactNative.Keyboard.addListener')

**Comment by swaroopa94** [comment]
I too Got the same error 
any solution?

**Comment by alvaromb** [comment]
Which RN version are you running? And which `react-native-keyboard-aware-scroll-view` version?

**Comment by swaroopa94** [comment]
"react-native": "^0.25.1",
  "react-native-keyboard-aware-scroll-view": "^0.2.1",

**Comment by alvaromb** [comment]
Check the supported versions in the readme: https://github.com/APSL/react-native-keyboard-aware-scroll-view#supported-versions

tl;dr

0.2.0 needs RN 0.32 or higher.

**Comment by swaroopa94** [comment]
It has been installed automatically i did not changed the version

**Comment by alvaromb** [comment]
You have to specify the supported version, but by default it will install the latest, it's how npm works.
